ubuntu2004/profiles/cis_level1_server.profile

```diff
@@ -0,0 +1,653 @@
+documentation_complete: true
+
+title: 'CIS Ubuntu 20.04 Level 1 Server Benchmark'
+
+description: |-
+ This baseline aligns to the Center for Internet Security
+ Ubuntu 20.04 LTS Benchmark, v1.0.0, released 07-21-2020.
+
+selections:
+    # 1 Initial Setup #
+    ## 1.1 Filesystem Configuration ## 
+    ### 1.1.1 Disable unused filesystems ###
+    #### 1.1.1.1 Ensure mounting of cramfs filesystems is disabled (Automated)
+    - kernel_module_cramfs_disabled
+
+    #### 1.1.1.2 Ensure mounting of freevxfs filesystems is disabled (Automated)
+    - kernel_module_freevxfs_disabled
+
+    #### 1.1.1.3 Ensure mounting of jffs2 filesystems is disabled (Automated)
+    - kernel_module_jffs2_disabled
+
+    #### 1.1.1.4 Ensure mounting of hfs filesystems is disabled (Automated)
+    - kernel_module_hfs_disabled
+
+    #### 1.1.1.5 Ensure mounting of hfsplus filesystems is disabled (Automated)
+    - kernel_module_hfsplus_disabled
+
+    #### 1.1.1.6 Ensure mounting of udf filesystems is disabled (Automated)
+    - kernel_module_udf_disabled
+
+    #### 1.1.1.7 Ensure mounting of FAT filesystems is limited (Manual)
+    # Skip due to being Level 2
+
+    ### 1.1.2 Ensure /tmp is configured (Automated)
+    - partition_for_tmp
+
+    ### 1.1.3 Ensure nodev option set on /tmp partition (Automated)
+    - mount_option_tmp_nodev
+
+    ### 1.1.4 Ensure nosuid option set on /tmp partition (Automated)
+    - mount_option_tmp_nosuid
+
+    ### 1.1.5 Ensure noexec option set on /tmp partition (Automated)
+    - mount_option_tmp_noexec
+
+    ### 1.1.6 Ensure /dev/shm is configured (Automated)
+    # Skip due to being handled by systemd and ensured by follow-on tests
```

Comment:
Yeah, it'd be nice to have an entry here for `/dev/shm` (and also `/tmp`) to have it configured in `systemd` correctly, rather than adding `/etc/fstab` changes.